core/runtime/src/main/java/io/quarkus/runtime/annotations/RunOnWorkerThread.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.runtime.annotations;
+
+import static java.lang.annotation.ElementType.*;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import javax.inject.Qualifier;
+
+/**
+ * Qualifier used to indicate that the annotated element should not be executed on the IO thread but on
+ * a worker thread.
+ * This qualifier is used to allow <em>blocking</em> processing in a reactive application.
+ * For example, use this annotation if your reactive route or gRPC service need to use blocking APIs such
+ * as <em>blocking</em> database access.
+ */
+@Qualifier
```

Comment:
Why a CDI qualifier?

Reply:
Because I copied the `@Blocking` annotation from REactive Messaging which is a qualifier ;-). In quarkus it's not required.

Reply:
Ok, then go and remove it ;-)

Reply:
Done.

Reply:
It still seems to be there